packages/nimble-components/src/select/styles.ts

```diff
@@ -1,6 +1,37 @@
 import { css } from '@microsoft/fast-element';
 import { styles as dropdownStyles } from '../patterns/dropdown/styles';
+import { styles as errorStyles } from '../patterns/error/styles';
+import { borderWidth } from '../theme-provider/design-tokens';
+import { appearanceBehavior } from '../utilities/style/appearance';
+import { DropdownAppearance } from './types';
 
 export const styles = css`
     ${dropdownStyles}
-`;
+    ${errorStyles}
```

Comment:
I'm seeing a double underline in storybook when I turn on the error state and then tab focus into the component.
<img width="271" alt="image" src="https://user-images.githubusercontent.com/10500124/196320006-e2b445cb-8fbb-421d-8e22-e7195c851f34.png">

I tried this with a few other inputs and didn't see the double underline so it seems wrong to me, though I didn't check the XD spec.

Reply:
This is by design. Inputs that immediately get the cursor on focus (e.g. text field, numeric field, combo box) don't get this additional focus indicator. However, controls that don't show a blinking cursor on focus (e.g. tabs, button, checkbox) get a double underline/outline. This is not a change made in this PR (except for turning the double underline red when in the invalid state).